index.html

```diff
@@ -48,9 +49,9 @@
         </ul>
       </nav>
     </header>
-    
+
     <section class="headline">
-      
+
       <h1 class="header-1 color-n800 mb-12">
         I’m Elmar,<br />Glad to see you!
       </h1>
```

Comment:
- [ ] **Please make sure that the projects are created dynamically using Javascript.**

In this milestone, the already created HTML code for the projects list should be removed, and instead, it should be added via JavaScript. Please create objects with each object containing information about your projects. Then use that array to dynamically create projects.

Reply:
Dear @smcommits,

I have fixed it. Please, check.

MR

Elmar